website/content/en/docs/tutorials/iperf.md

```diff
@@ -199,37 +254,147 @@ brew link --force gettext
 The `CLIENT_IP` and `SERVER_IP` refers to the node IP of the peer kind cluster, which assigns the peer YAML file.
 {{< /notice >}}
 
-2. In the server cluster, export the iperf3-server service:
+### Export setup
```

Comment:
nit: since these show as headers and in ToC, maybe better to use more complete sentences
```suggestion
### Export iPerf server endpoint
```